.circleci/config.yml

```diff
@@ -265,6 +277,15 @@ workflows:
             branches:
               only: master
 
+      - docker_push_tag:
+          requires:
+            - build_check
```

Comment:
This dependancy should be either removed or the same  tags filter should be added to it.
Not sure what's the value of `CIRCLE_BRANCH` during a tagging in `build_check` job